### ISSUE TITLE
Does not install

### ISSUE THREAD

**Issue by nedelman** [SmokeMonsterPacks/Super-NT-Jailbreak#7]
I downloaded the .bin file, and saved it to the root directory of a freshly formatted 32GB SD card. I formatted the card using the "SD Card Formatter" application from www.sdcard.org.

After inserting the SD card into the Super NT and powering the system on, the power LED flashes red with an apparent pattern. The pattern seems to be "4 flashes, 2 flashes", which then repeats over and over. The screen never boots up while this is happening.

I am running the latest 4.1 firmware on the Super NT.

**Comment by jan-kleks** [comment]
What type of an SD card is this?

**Comment by nedelman** [comment]
Samsung EVO micro SD in a full SD adapter. The exact same brand/type I'm using in my NT Mini.

**Comment by jan-kleks** [comment]
@nedelman Can [this](http://atariage.com/forums/topic/242970-fpga-based-videogame-system/?p=3962714) be the problem?

**Comment by nedelman** [comment]
I'll try again and see what happens if I just leave it for a longer amount of time.

**Comment by jan-kleks** [comment]
@nedelman Just don't forget to report back.

**Comment by SmokeMonsterPacks** [comment]
Make sure to give it a few minutes to flash. I've just added a note about this in the README.

**Comment by nedelman** [comment]
I left it for about 15 minutes and there was no change. The flashing pattern was the same as before.

And just to be more explicity about my SD card's setup:
-The card is formatted as FAT32 (as instructed)
-The only file in the card's root directory is the .bin file (as instructed)
-The only directory on the card is "SNES"

**Comment by SmokeMonsterPacks** [comment]
Could you please try formatting the card following these instructions? http://www.retrorgb.com/romcarts.html#format

If that works, I can point people there for a solution.

**Comment by jan-kleks** [comment]
@nedelman If the @SmokeMonsterPacks' solution does not work you can also try to use the Overwrite format option of SD Card Formatter.

**Comment by nedelman** [comment]
I figured out the problem, and it's a bit dumb, but may help others in the future:

Basically, within Chrome I was right clicking on the .bin file on the Github page and selecting "Save Link As..." which is actually saving the page, not the .bin file. After going to the page and then correctly saving the .bin file, everything seems to be working.

Also I checked out the retrorgb.com link, and the "SD Card Formatter" screenshot is outdated. The most recent version (5.0) does not look like that or have the options shown in the screenshots.
